### PR TITLE
Style/kal input

### PR DESCRIPTION
**Pull request by KALjpo**

### CODE REVIEW

**Review by jsp-up — APPROVED** [review]

File: projects/kalidea/kaligraphi/src/lib/atoms/kal-input/kal-input.component.sass [hunk 1]
```diff
@@ -1,5 +1,65 @@
...
+        // Status
+        &:active, &:focus
+          border-color: get("palettes.grey.wf200", $kal-parameters)
+        &:disabled
+          color: get("palettes.grey.wf700", $kal-parameters)
+          + kal-icon
+            color: get("palettes.grey.wf700", $kal-parameters)
+            cursor: none
+        &.ng-invalid
+          border-color: get("palettes.red.normal", $kal-parameters)
```
Comment (ydomenjoud) [inline projects/kalidea/kaligraphi/src/lib/atoms/kal-input/kal-input.component.sass:60]: same as default, should be removed

File: projects/kalidea/kaligraphi/src/lib/atoms/kal-input/kal-input.component.sass [hunk 2]
```diff
@@ -1,5 +1,65 @@
...
+          right: 12px
+          font-size: 80%
+          cursor: pointer
+
+    &.kal-reverse
+
+      div > input
+        color: get("palettes.grey.wf800", $kal-parameters)
+        background-color: get("palettes.grey.wf500", $kal-parameters)
+        border-width: 1px
```
Comment (ydomenjoud) [inline projects/kalidea/kaligraphi/src/lib/atoms/kal-input/kal-input.component.sass:47]: same as default, should be removed

File: projects/kalidea/kaligraphi/src/lib/atoms/kal-input/kal-input.component.sass [hunk 3]
```diff
@@ -1,5 +1,65 @@
...
+        border-style: solid
```
Comment (ydomenjoud) [inline projects/kalidea/kaligraphi/src/lib/atoms/kal-input/kal-input.component.sass:48]: same as default, should be removed

File: projects/kalidea/kaligraphi/src/lib/atoms/kal-input/kal-input.component.sass [hunk 4]
```diff
@@ -1,5 +1,65 @@
...
+        border-color: get("palettes.grey.wf500", $kal-parameters)
+        border-radius: $base-border-radius
```
Comment (ydomenjoud) [inline projects/kalidea/kaligraphi/src/lib/atoms/kal-input/kal-input.component.sass:50]: same as default, should be removed

**Pull request merged**